packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -134,6 +134,145 @@ describe("<DateRangePicker>", () => {
         });
     });
 
+    describe("left/right calendar when not sequential", () => {
+        it("only shows one calendar when minDate and maxDate are the same month", () => {
+            const isSequential = false;
+            const minDate = new Date(2015, Months.DECEMBER, 1);
+            const maxDate = new Date(2015, Months.DECEMBER, 15);
+
+            renderDateRangePicker({ isSequential, maxDate, minDate });
+            assert.lengthOf(document.getElementsByClassName("DayPicker"), 1);
+            assert.lengthOf(document.getElementsByClassName(".DayPicker-NavButton--prev"), 0);
+            assert.lengthOf(document.getElementsByClassName(".DayPicker-NavButton--next"), 0);
+        });
+
+        it("left calendar bound between minDate and (maxDate - 1 month)", () => {
```

Comment:
Would prefer to have a verb somewhere in this title; the goal of the test isn't crystal clear at a glance.